integrations/oci/tls-certificates/src/test/java/io/helidon/integrations/oci/tls/certificates/TestOciCertificatesDownloader.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.integrations.oci.tls.certificates;
+
+import java.io.InputStream;
+import java.security.cert.Certificate;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+
+import io.helidon.common.Weight;
+import io.helidon.common.Weighted;
+import io.helidon.integrations.oci.tls.certificates.spi.OciCertificatesDownloader;
+
+import jakarta.inject.Singleton;
+
+@Singleton
+@Weight(Weighted.DEFAULT_WEIGHT + 1)
+class TestOciCertificatesDownloader extends DefaultOciCertificatesDownloader {
```

Comment:
The usual naming convention is that test classes END with the `Test` word, and test methods start with `test` word. Please fix your tests and change your template.

Reply:
This is not a Junit Test.  It is a Fake downloader impl with a higher weight then the OCI one.